cfgov/tccp/jinja2/tccp/includes/data_published.html

```diff
@@ -1,11 +1,16 @@
 {%- import 'v1/includes/macros/time.html' as time -%}
 
-{%- macro data_published(date) -%}
+{%- macro data_published(date, verbose=false) -%}
 
-<p style="font-size: 14px; color: #5a5d61">
-    Data published {{ time.render(
+<p class="u-small-text u-small-text--gray">
```

Comment:
It might be better to create a general color utility class, `u-color-gray`, that sets the color to gray. Or change the modifier name to specify the semantic meaning for the `u-small-text` modifier. Why is the small text gray? Maybe something like `u-small-text--subtle` might make more semantic sense.

Reply:
@anselmbradford, I went with the more semantic `u-small-text--subtle` in 1a60984.